src/backend/base/langflow/base/data/base_file.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, *args, **kwargs):
             ),
             required=False,
             input_types=["Data", "Message"],
-            is_list=True,
+            list=True,
```

Comment:
Hi, @erichare, May I know why do we need to change is_list=True to list=True?
I belive the correct way of using it is is_list=True,

```suggestion
            is_list=True,
```

Reply:
@edwinjosechittilappilly i was getting a complaint from mypy regarding `is_list` not being a real parameter. that it was changed to `list` - am i incorrect? I very well could be, i thought maybe the parameter had a name change at some point

Reply:
I think this should be fine too since is_list have list as alias in `ListableInputMixin`

Reply:
The alias is because the frontend uses `list` but `is_list` is the correct parameter.